setup.py

```diff
@@ -28,6 +28,7 @@ def get_version(rel_path):
     "Jinja2>=2.8,<3",
     "colorama>=0.3.9",
     "packaging>=16.8,<17.0",
+    "sceptre-file-resolver>=1.1.2,<2",
```

Comment:
this is not the cmd resolver

Reply:
Woops. I updated the version but forgot the name. Fixed.